dvc/tree/osf.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import os
+import threading
+
+from funcy import cached_property, wrap_prop
+
+from dvc.exceptions import DvcException
+from dvc.hash_info import HashInfo
+from dvc.path_info import URLInfo
+from dvc.progress import Tqdm
+from dvc.scheme import Schemes
+
+from .base import BaseTree
+
+logger = logging.getLogger(__name__)
+
+
+class OSFAuthError(DvcException):
+    def __init__(self):
+        message = (
+            "OSF authorization failed. Please check or provide"
+            " osf_username and password. See "
+            "https://bit.ly/375On32 for details"
```

Comment:
hmm, could we use the dvc.org/doc troubleshooting mechanism here? bitly in the exceptions looks a bit strange, probably we don't want to depend on bitly

Reply:
Can I see somewhere an example of how to use the dvc.org/doc troubleshooting mechanism? Does it somehow allow you not to use a long link (like https://dvc.org/doc/command-reference/remote) in a error message?

Reply:
yep, please try to grep in sources links like `error.dvc.org` . E.g. `https://error.dvc.org/missing-files` goes to the docs page. You can make a section on that page that provides more details, additional links, etc. 

Or in this specific case we might just provide a link to the `dvc remote modify` page? It'll have the OSF section?

Reply:
Thank you for the advice, I'll see what needs to be done.

Reply:
Yes, I opened PR with changes in docs and there will be a section for OSF on `dvc remote modify` page.